web3/process.go

```diff
@@ -475,6 +500,66 @@ func (c *Contracts) MonitorProcessStateRootChange(ctx context.Context, interval
 	return updatedProcChan, nil
 }
 
+// MonitorProcessMaxVotersChange monitors the max voters changes in processes
+// by polling the ProcessRegistry contract every interval. It returns a channel
+// that emits processes with the new max voters.
+func (c *Contracts) MonitorProcessMaxVotersChange(ctx context.Context, interval time.Duration) (<-chan *types.ProcessWithMaxVotersChange, error) {
```

Comment:
Why do we need a new independent monitor to track the maxvoters change? Cannot we do it in the existing that updates the Process details? 

I may be wrong, but I think we should try to avoid having multiple processes doing the same. Instead, it'd make more sense to have a single ProcessMonitor that updates everything at once.